aptos-move/aptos-vm/src/errors.rs

```diff
@@ -26,6 +26,8 @@ pub const ESCRIPT_NOT_ALLOWED: u64 = 1008;
 pub const EMODULE_NOT_ALLOWED: u64 = 1009;
 pub const EINVALID_WRITESET_SENDER: u64 = 1010; // invalid sender (not aptos root) for write set
 pub const ESEQUENCE_NUMBER_TOO_BIG: u64 = 1011;
+// This error code has been deprecated and just left here sp future error codes would not
+// accidentally reuse.
 pub const EBAD_TRANSACTION_FEE_CURRENCY: u64 = 1012;
```

Comment:
Maybe we should rename this to RESERVED_UNUSED or something?

Reply:
Updated